model/repository.go

```diff
@@ -49,9 +50,16 @@ type Revision struct {
 }
 
 type GitTag struct {
-	Tag             string
-	Pusher          string
-	PusherGithubUID int
+	Tag    string
+	Pusher string
+}
+
+func FormatGitTagsAsString(tags []GitTag) string {
```

Comment:
Could you create a String() method on GitTag, then join those in this function? That way any other displays which user the stringer interface will know what to display for a git tag